modules/android/pages/_partials/pn-issues-list.adoc

```diff
@@ -1,6 +1,47 @@
 // BEGIN: tagged-inclusion -- issues list for Android
 == Issues in Release 2.8
 
+
+// BEGIN 2-8-6
+
+=== 2.8.6
+
+// tag::issues-2-8-6[]
+// ==== Deprecated
+// tag::deprecated-2-8-6[]
+// end::deprecated-2-8-6[]
+
+==== Enhancements
+// tag::enhancements-2-8-6[]
+include::{root-partials}pn-issues-list-litecore.adoc[tag=enhancements-2-8-6]
+
+* https://issues.couchbase.com/browse/CBL-1979[CBL-1979] -- Android v30 compatibility
+* https://issues.couchbase.com/browse/CBL-2001[CBL-2001] -- Android v30 compatibility
+
+// end::enhancements-2-8-6[]
+
+
+==== Fixed
+// tag::fixed-2-8-6[]
+include::{root-partials}pn-issues-list-litecore.adoc[tag=fixed-2-8-6]
+
+* https://issues.couchbase.com/browse/CBL-1984[CBL-1984] -- Support domain specific server authentication
+* https://issues.couchbase.com/browse/CBL-2037[CBL-2037] -- Core pool size is too small
+* https://issues.couchbase.com/browse/CBL-2089[CBL-2089] -- Replication failed to establish to a sync gateway configured with a system-trusted CA cert
```

Comment:
This turned out to be an internal setup error, fixed by IT. Do not include.

Reply:
Deleted CBL-2089